src/input_common/helpers/stick_from_buttons.cpp

```diff
@@ -11,6 +11,9 @@ namespace InputCommon {
 
 class Stick final : public Common::Input::InputDevice {
 public:
+    // Some games don't like the theorical maximum so aim for one unit off
```

Comment:
Documents the game affected and the rationale behind why one unit lower is used
```suggestion
    // Some games such as EARTH DEFENSE FORCE: WORLD BROTHERS
    // do not play nicely with the theoretical maximum range.
    // Using a value one lower from the maximum emulates real stick behavior.
```

Reply:
Also modifier scale is not changed? Perhaps you should just std::clamp the value

Reply:
Modifier scale is by default 50%. The idea is not to completely prevent this from happening. Just making it unlikely to happen. As the maximum value is completely valid. Just not for this particular game.